src/oci-umount.c

```diff
@@ -592,20 +592,20 @@ static int prestart(
 	for (i = 0; i < nr_umounts; i++) {
 		nr_mapped = map_mount_host_to_container(id, config_mounts, config_mounts_len, mounts_on_host[i].path, mapped_paths, MAX_MAPS);
 		if (nr_mapped < 0) {
-			pr_perror("%s: Error while trying to map mount [%s] from host to conatiner. Skipping.", id, mounts_on_host[i].path);
+			pr_perror("%s: Error while trying to map mount [%s] from host to container. Skipping.", id, mounts_on_host[i].path);
 			continue;
 		}
 
 		if (!nr_mapped) {
-			pr_pinfo("%s: Could not find mapping for mount [%s] from host to conatiner. Skipping.", id, mounts_on_host[i].path);
+			pr_pdebug("%s: Could not find mapping for mount [%s] from host to container. Skipping.", id, mounts_on_host[i].path);
 			continue;
 		}
 
 		for (int j = 0; j < nr_mapped; j++) {
 			snprintf(umount_path, PATH_MAX, "%s%s", rootfs, mapped_paths[j]);
 			ret = unmount(id, umount_path, mounts_on_host[i].submounts_only, mnt_table, mnt_table_sz);
 			if (ret < 0) {
-				pr_perror("%s: Skipping unmount path: [%s]", id, umount_path);
+				qpr_perror("%s: Skipping unmount path: [%s]", id, umount_path);
```

Comment:
You mean `pr_error()` instead of `qpr_error()` ? This probably does not compile.